src/vscode-azuredatabases.api.d.ts

```diff
@@ -3,39 +3,39 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-export interface CosmosDBExtensionApi {
+export interface AzureDatabasesExtensionApi {
```

Comment:
> Should the API name vscode-cosmosdbgraph.api change?

No - I would not change this yet (or anything in this file). It is an internal detail that has no affect on our branding as far as the user is concerned. We will have to thoughtfully refactor this API to also support Postgres and I would prefer to change this file at that point